platform/ios/platform/ios/src/MGLMapView.mm

```diff
@@ -2591,8 +2586,7 @@ - (MGLMapCamera *)cameraByZoomingToZoomLevel:(double)zoom aroundAnchorPoint:(CGP
 
 - (MGLMapCamera *)cameraByRotatingToDirection:(CLLocationDirection)degrees aroundAnchorPoint:(CGPoint)anchorPoint
 {
-    mbgl::EdgeInsets padding = MGLEdgeInsetsFromNSEdgeInsets(self.contentInset);
-    mbgl::CameraOptions currentCameraOptions = self.mbglMap.getCameraOptions(padding);
+    mbgl::CameraOptions currentCameraOptions = self.mbglMap.getCameraOptions();
```

Comment:
Here and in similar places below the padding value passed in `mbglMap.getCameraOptions()` just replaces the camera edge insets. Although `CameraOptions` is a pure model class which doesn't have any computations and side effects, it doesn't make much sense to replace its padding value.